client/src/modules/email-report/index.html

```diff
@@ -0,0 +1,217 @@
+                  class="form-control">
+                    <option  
+                      ng-repeat="item in EmailReportCtrl.frequencies"
+                      value="{{item}}">
+                        {{item}}
+                    </option>
+                </select>
+                 <div class="help-block"
+                  ng-messages="EmailReportRegistrationForm.frequency.$error"
+                  ng-show="EmailReportRegistrationForm.$submitted">
+                    <div ng-messages-include="modules/templates/messages.tmpl.html">
+                    </div>
+                </div>
+
+              </div>
+            </div>
+
+
+            <!-- report Group -->
+
+            <div
+              class="form-group"
+              ng-class="{'has-error' : EmailReportRegistrationForm.code_report_group.$invalid && EmailReportRegistrationForm.$submitted}">
+
+              <label class="control-label ng-binding" translate>
+                FORM.LABELS.REPORT_GROUP
+              </label>
+
+              <div class="input-group col-lg-12">
+                <select 
+                name='code_report_group'
+                  required=""
+                  ng-model="EmailReportCtrl.emailReport.code_report_group"
+                  class="form-control">
+                    <option 
+                      ng-repeat="item in EmailReportCtrl.report_groups"  
+                      value="{{item.code}}">
+                        {{item.name}}
+                    </option>
+                </select>
+                <div class="help-block"
+                  ng-messages="EmailReportRegistrationForm.code_report_group.$error"
+                  ng-show="EmailReportRegistrationForm.$submitted">
+                    <div ng-messages-include="modules/templates/messages.tmpl.html">
+                    </div>
+                </div>
+              </div>
+            </div>
+
+
+
+            <bh-loading-button 
+             ng-show="EmailReportCtrl.selectedEmailReport.selected==false"
+             class="pull-right" loading-state="BedRegistrationForm.$loading">
+              {{ "FORM.BUTTONS.SAVE" | translate }}
+            </bh-loading-button>
+
+            <span class="btn btn-danger" 
+              ng-click="EmailReportCtrl.remove(EmailReportRegistrationForm)"
+              ng-show="EmailReportCtrl.selectedEmailReport.selected==true"
+              class="pull-right">
+                {{ "FORM.BUTTONS.DELETE" | translate }}
+            </span>
+
+            <bh-loading-button
+              ng-show="EmailReportCtrl.selectedEmailReport.selected==true"
+              class="pull-right"
+              loading-state="BedRegistrationForm.$loading">
+                {{ "FORM.BUTTONS.UPDATE" | translate }}
+            </bh-loading-button>
+
+        </form>
+
+      </div>
+      <!-- end new Reporting group form -->
+
+      <!-- start grid of emails -->
+
+      <div class="col-lg-7">
```

Comment:
The rest of the application uses `col-md-*`.  To make sure everything works the same way across multiple screen sizes, you should use `col-md-7`.

You should also update all places where this is used.